src/test/java/net/masterthought/cucumber/ReportBuilderTest.java

```diff
@@ -99,10 +101,10 @@ public void generateReports_GeneratesPages() {
     public void generateReports_WithTrendsFile_GeneratesPages() {
 
         // given
-        List<String> jsonReports = Arrays.asList(ReportGenerator.reportFromResource(ReportGenerator.SAMPLE_JSON));
+        List<String> jsonReports = Arrays.asList(ReportGenerator.reportFromResourceJson(ReportGenerator.SAMPLE_JSON));
```

Comment:
if you revert to old constructor then most or maybe all existing tests could be left unchanged

Reply:
yeah as per suggestion above I'll do that update